airflow/example_dags/example_xcom.py

```diff
@@ -43,24 +44,14 @@ def push_by_returning(**kwargs):
     return value_2
 
 
-def puller(**kwargs):
+def puller(pulled_value_1, pulled_value_2, **kwargs):
     """Pull all previously pushed XComs and check if the pushed values match the pulled values."""
-    ti = kwargs['ti']
 
-    # get value_1
-    pulled_value_1 = ti.xcom_pull(key=None, task_ids='push')
+    # check pulled value 1
     if pulled_value_1 != value_1:
         raise ValueError(f'The two values differ {pulled_value_1} and {value_1}')
 
-    # get value_2
-    pulled_value_2 = ti.xcom_pull(task_ids='push_by_returning')
-    if pulled_value_2 != value_2:
-        raise ValueError(f'The two values differ {pulled_value_2} and {value_2}')
-
-    # get both value_1 and value_2
-    pulled_value_1, pulled_value_2 = ti.xcom_pull(key=None, task_ids=['push', 'push_by_returning'])
-    if pulled_value_1 != value_1:
-        raise ValueError(f'The two values differ {pulled_value_1} and {value_1}')
```

Comment:
I would still keep an example of accessing `ti` from `kwargs` for example:
```
# check pulled value 1
...
# same using explicit xcom pull
from_xcom = ti.xcom_pull(key=None, task_ids='push')
if from_xcom != pulled_value_1:
    raise ValueError(f'The two values differ {pulled_value_1} and {value_1}
```
WDYT?

Reply:
I do agree. Will fix it.